numpy/core/shape_base.py

```diff
@@ -304,6 +304,11 @@ def stack(arrays, axis=0):
     See Also
     --------
     concatenate : Join a sequence of arrays along an existing axis.
+    vstack : Stack arrays in sequence vertically, along the existing row axis.
```

Comment:
You're right about 'in sequence', I'll remove it. There is only one 'row axis' though?